backend/configs/usr/lib/wb-homeui-backend/fcgiwrap_prestart.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# nginx user should has rw access to fwupdate uploads dir
+nginx_uploads_dir="/mnt/data/uploads"
+nginx_user="www-data"
+if [[ -d $nginx_uploads_dir ]] && $(id -u $nginx_user > /dev/null 2>&1); then
```

Shellcheck (0.10.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)